lms/djangoapps/static_template_view/views.py

```diff
@@ -54,6 +56,13 @@ def render(request, template):
         # This is necessary for the dialog presented with the TOS in /register
         if template == 'honor.html':
             context['allow_iframing'] = True
+        elif template == 'ferie_bez_nudy.html':
```

Comment:
To jest kompletnie zbędne; ten widok jest nieużywany

Reply:
@nazgut możesz się odnieść to tego ? bo to Twoje :)